app/assets/styles/styles.css

```diff
@@ -170,3 +170,137 @@ a {
     font-size: 1.25rem;
   }
 }
+
+.teaser {
+  font-family: 'Trajan Pro Regular', 'Times New Roman', serif;
```

Comment:
If we set the custom font to apply to the whole section we have to manually set each time we want to use the default body font (Roboto). Since we use Trajan on select elements only, it is cleaner to declare the font-family on those elements only, for example, the teaser section title _Our News_ and the teaser name.

Let's remove L175 and only apply Trajan on the elements were we want to change the font from the body font. Then you can remove all lines of code where you apply Roboto to the element.